install/kubernetes/README.md

```diff
@@ -45,12 +45,12 @@ Helm chart for Tetragon
 | tetragon.argsOverride | list | `[]` |  |
 | tetragon.btf | string | `""` |  |
 | tetragon.commandOverride | list | `[]` |  |
-| tetragon.enableCiliumAPI | bool | `true` |  |
+| tetragon.enableCiliumAPI | bool | `false` |  |
```

Comment:
Is there a reason we want this disabled by default?

Reply:
right now i think it's mostly optimized for quickstart guides, we don't want to require cilium for quickstart to work.